serialization/pickle.ksy

```diff
@@ -0,0 +1,443 @@
+meta:
+  id: pickle
+  title: Python pickle serialiazation format (protocol 3)
+  application: Python
+  file-extension:
+    - pickle
+    - pkl
+    #- pkl0
```

Comment:
Why not?

Reply:
Sorry, I meant to take those out. `.pkl[0-4]` are some personal conventions I use. They aren't adopted anywhere else that I know of.